packages/flutter_tools/lib/src/build_runner/resident_web_runner.dart

```diff
@@ -271,16 +281,10 @@ abstract class ResidentWebRunner extends ResidentRunner {
   @override
   Future<void> debugToggleDebugPaintSizeEnabled() async {
     try {
-      final vmservice.Response response =
-          await _vmService?.callServiceExtension(
-        'ext.flutter.debugPaint',
-      );
-      await _vmService?.callServiceExtension(
-        'ext.flutter.debugPaint',
-        args: <dynamic, dynamic>{
-          'enabled': !(response.json['enabled'] == 'true')
```

Comment:
Your implementation is a purer "toggle" but it looks like it wasn't really toggling before, it was really checking `enabled`.  Does this matter?  Is there anywhere where it's being set multiple times, knowing `enabled` is staying the same?

Reply:
Whats going on here is that the first response reads the state of `debugPaint` as `enabled` and the second updates the state of `debugPaint` to the opposite of that value.